### ISSUE TITLE
Player is missing volume control

### ISSUE THREAD

**Issue by nkkollaw** [elementary/videos#51]
I like minimalism, but a player without a volume control is kind of useless.

Sure, I can control the whole PC's volume, but is that what I want to do..?

**Comment by tintou** [comment]
Yes, it is what you want to do, or are you mixing the sound of several audio sources by hand ?

**Comment by nkkollaw** [comment]
I often have music and porn movies on I don't want the whole house to hear.

VLC is fine, I'm just wondering why go over the top with removing features people actually want, I would have happily kept Videos.

**Comment by aleksandar-stefanovic** [comment]
There's an built-in sound mixer, if you want to mix multiple sources:
![screenshot from 2017-12-14 10 25 40](https://user-images.githubusercontent.com/6058082/33984821-2c73d1a6-e0b9-11e7-827e-2f7bf3806fd7.png)

It is located in Settings > Sound.

**Comment by nkkollaw** [comment]
@aleksandar-stefanovic : thanks for your help. I do appreciate it but it doesn't look very convenient vs. a volume control like all other players.

Still think it's a design flaw.

**Comment by murermader** [comment]
Too bad the sound mixer is not existent on elementaryOS 5.1 Hera. The "output volume" settings also does NOT affect videos. Sometimes I just have to shake my head at elementaryOS, just for being different for the sake of being different, not for the sake of actually being usable.

**Comment by jeremypw** [comment]
I install PulseAudio Volume Control from the AppCenter to make up for some of the native sound control deficiencies.  Not ideal - hopefully native sound control will catch up.

**Comment by seesemichaelj** [comment]
Another use case: Sometimes the volume of the video I'm playing is very low. So even at max system volume, I'd like to boost the volume even further; this is where VLC comes in and allows positive gain (up to 125%). Perhaps it's not necessary for most users and that may be ok, but I just end up not using the built in video player in favor of VLC.

To play devil's advocate though, I hate it when the VLC volume is some random thing from when I adjusted it and my OCD wants to set it at 100%.

**Comment by ShaminAsfaq** [comment]
I would love to use this as my default player. But with volume control missing, I had to install SMPlayer. For some reason, VLC lags on my machine while seeking video files forward or backward. Hence, I didn't install it on my new Hera install yesterday.
